### ISSUE TITLE
[bug] vuex store presence is not detected if not using `.js` file extension

### ISSUE THREAD

**Issue by huafu** [quasarframework/quasar-cli#135]
### Software version
quasar-cli: 0.16.4
OS: macOS 10.13.4 (17E202)
Node: v10.3.0
NPM: 6.1.0
**I am using TypeScript, so `.ts` file extensions**

### What did you get as the error?
The store is not recognized

### What were you expecting?
The store should be detected and included in the `app` object when booting the root view

### What steps did you take, to get the error?
Created a typescript project with Quasar cli (including changing the `quasar.conf.js`, especially the new `sourceFiles` options: `sourceFiles.store: 'src/store/index.ts'`)

### How I fixed it:
I got it fixed by patching the `QuasarConfig` class (with `require('quasar-cli/lib/quasar-config')`, wrapping its `compile` method and setting `<QuasarConfig>this.buildConfig.store = true` right after the `compile` method has been called.

**Comment by huafu** [comment]
For those interested, as this might be also the case with CoffeeScript, here is what I have at the top of my `quasar.conf.js`:

```js
const QuasarConfig = require('quasar-cli/lib/quasar-config')

QuasarConfig.prototype.compile = (function(original) {
  return function() {
    const result = original.call(this, ...arguments)
    this.buildConfig.store = true
    return result
  }
})(QuasarConfig.prototype.compile)
```

**Comment by rstoenescu** [comment]
Available in future CLI v0.16.5.

**Comment by huafu** [comment]
@rstoenescu you rocks! 👍
